src/marionette.sliding-view.js

```diff
@@ -76,7 +81,7 @@ Mn.SlidingView = Mn.CollectionView.extend({
     }
 
     // If the boundaries are unchanged, then we bail out early
-    if (lowerBound === this._lowerBound && upperBound === this._upperBound) {
+    if (!this.forceUpdate && (lowerBound === this._lowerBound && upperBound === this._upperBound)) {
```

Comment:
This conditional has gotten a little out of hand. Maybe...

```js
var boundsUpdated = lowerBound !== this._lowerBound && upperBound !== this._upperBound;

if (!this.forceUpdate && !boundsUpdated) {
  // toop toop
}
```

also, could you update the comment?